### DISCUSSION TITLE
Gridding with scatter points

### DISCUSSION THREAD

**Discussion by oyvindi** [MaxRev-Dev/gdal.netcore#146] [Q&A] [answered]
I'm trying to created a grid based on a memory dataset and point geometries, but have a hard time to figure it out in the C# binding.

I assume the Gdal.wrapper_GDALGrid() can be used for this. How is this used, and how do I pass the options for this, such as algorithm etc ?

**Comment by MaxRev-Dev** [comment, marked as answer]
You need to rasterize the scatered points and then use the gdal_grid wrapper on the raster dataset.
Check the documentation on these functions or try asking ChatGPT.
